pkg/cache/cache.go

```diff
@@ -601,9 +601,9 @@ func (c *Cache) Usage(cqObj *kueue.ClusterQueue) (*ClusterQueueUsageStats, error
 	}
 
 	stats := &ClusterQueueUsageStats{
-		ReservedResources:  getUsage(cq.Usage, cq.ResourceGroups, cq.Cohort),
+		ReservedResources:  getUsage(cq.Usage, cq, cq.Cohort),
```

Comment:
```suggestion
		ReservedResources:  getUsage(cq),
```
Since the other fields also go with CQ, can be a follow up

Reply:
though this function is called again below with `AdmittedUsage`

Reply:
ah, got it, we could probably still get rid of the cohort at least, and discrimanate usage / admitted usage with a bool flag. Passing the same information in two parameters is a bit awkward imo, but I'm ok to leave it for follow up.